packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.ts

```diff
@@ -100,7 +100,7 @@ export const createAnnouncmentString = (
    * Filter participants into two arrays to put all the unnamed participants at the back of the
    * names being announced.
    */
-  const unnamedParticipants = participants.filter((p) => p.displayName === undefined);
+  const unnamedParticipants = participants.filter((p) => !p.displayName);
```

Comment:
How does this also cover the case when the `displayName === ''`?

Reply:
This might feel a bit strange, but in js  ```!'' = true```

Reply:
@edwardlee-msft 's comment above explains this in a more detailed way 👍